cmd/main.go

```diff
@@ -40,9 +40,15 @@ func main() {
 			Usage: "watch for an address using utxo scanner",
 			Flags: []cli.Flag{
 				&cli.StringFlag{
-					Name:    "address",
-					Usage:   "address to watch",
-					Aliases: []string{"a"},
+					Name:     "address",
+					Usage:    "address to watch",
+					Aliases:  []string{"a"},
+					Required: true,
+				},
+				&cli.StringSliceFlag{
+					Name:     "peers",
```

Comment:
personal preference would use `connect` and singular, since we repeat `--connect` multiple time to have multiple results